system_tests/monitoring.py

```diff
@@ -181,6 +183,53 @@ def test_create_and_delete_metric_descriptor(self):
 
         retry_500(descriptor.create)()
         retry_404_500(descriptor.delete)()
+        descriptor.create()
+        retry_404(descriptor.delete)()
+
+        with self.assertRaises(NotFound):
+            descriptor.delete()
```

Comment:
I'd prefer to have any changes to `test_create_and_delete_metric_descriptor` in a separate pull request.